src/app/content/components/BookStyles.tsx

```diff
@@ -8,4 +8,8 @@ const Hoc = ({children, className}: {className?: string, children: (className?:
 
 export default styled(Hoc)`
   ${bookStyles}
+  /* Occur later so the bookstyles do not bleed below (e.g. @page) */
+  *:target {
```

Comment:
i have experienced problems with adding additional styles to the same component as the `bookStyles`, thats why i added some patchup styles in `ContentPane`

Reply:
Thanks! I moved them over & left a comment in here